### PR TITLE
Retemplate for 2.0

### PR DESCRIPTION
**Pull request by Cadair**
This adopts the new package template and also turns on isort.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ To push any changes you make to your github account by doing::
    $ git push origin branch-name
 
 where ``branch-name`` is the name of the branch you're working on.  Then
-from your github account you can request your changes to be merged to
+
```
Comment (nabobalis) [inline README.rst:94]:
```suggestion
from your github account you can request your changes to be merged to
```

**Comment by nabobalis** [comment]
```python
_________________________________________ ERROR collecting .tox/py38-devdeps/lib/python3.8/site-packages/ndcube/tests/test_ndcollection.py _________________________________________
../../.tox/py38-devdeps/lib/python3.8/site-packages/ndcube/tests/test_ndcollection.py:41: in <module>
    cube_collection = NDCollection([("cube0", cube0), ("cube1", cube1), ("cube2", cube2)], aligned_axes)
../../.tox/py38-devdeps/lib/python3.8/site-packages/ndcube/ndcollection.py:59: in __init__
    aligned_axes = collection_utils._sanitize_aligned_axes(keys, data, aligned_axes)
E   NameError: name 'collection_utils' is not defined
```
Or was that the last commit?

**Comment by Cadair** [comment]
I force pushed and fixed that.

**Comment by Cadair** [comment]
I merged this by mistake.

**Pull request closed without merging**